Sample-01/src/views/__tests__/ExternalApi.test.js

```diff
@@ -2,28 +2,33 @@ import React from "react";
 import { ExternalApiComponent } from "../ExternalApi";
 import { fireEvent, render, screen, waitFor } from "@testing-library/react";
 import "@testing-library/jest-dom/extend-expect";
+import { useAuth0 } from "@auth0/auth0-react";
+import { getConfig } from "../../config";
 
-jest.mock("@auth0/auth0-react", () => ({
-  useAuth0: jest.fn(() => ({
-    isLoading: false,
-    isAuthenticated: true,
-    getAccessTokenSilently: jest.fn(() => Promise.resolve("access-token")),
-  })),
-  withAuthenticationRequired: jest.fn(),
-}));
```

Comment:
This change is a pattern across some test modules. Upgrading to `react-scripts@4` sets Jest's `resetMocks` global setting to `true` by default, meaning that these mocks no longer work.

I could have just set the setting back to `false`, but the tests feel better with this change anyway.